### ISSUE TITLE
Horse Issue

### ISSUE THREAD

**Issue by Zestzy** [Wynncraft/Issues#8691]
I recently was farming in a cave, and while picking up items I guess I accidentally dropped my horse saddle without realizing. I am not asking for a black horse refund but just maybe something simple like a "are you sure you want to drop this horse" so you have to press q twice. Maybe even code a new slot for a horse so it cannot be dropped so easily. (It wouldn't be hard at all to code a /h or /callhorse either). 

Basically my point is create a more secure area to keep important items such as your horse, and attacking items like you already have for rings, and bracelets...

Please take this into consideration since it wouldn't be a hard fix, and would do a lot of good to players on the server who have also had this issue.

**Comment by Villageacules** [comment]
I think this is a great idea! An item lock is already a popular concept that the admins and developers know should be implemented. An item lock system is definitely in the future, it is one of the most suggested features on the server.
